cmd/rtcd/main.go

```diff
@@ -39,9 +41,25 @@ func main() {
 
 	logger.Info("rtcd: starting up")
 
+	service, err := service.New(cfg.Service, logger)
+	if err != nil {
+		logger.Error("rtcd: failed to create service", mlog.Err(err))
+		return
+	}
+
+	if err := service.Start(); err != nil {
```

Comment:
I know it's annoying, but are we ok with shadowing? (I am, personally, but depends on how we set up golangci-lint)

Reply:
I am generally okay with shadowing cause it keeps code simpler and I prefer to have a tight scope on the error variable, especially in cases where you don't necessarily return right away. By the way this reminds me I should have included `.golangci.yml`. Will make an extra push.